sqlglot/expressions.py

```diff
@@ -1351,6 +1355,10 @@ class GeneratedAsIdentityColumnConstraint(ColumnConstraintKind):
     }
 
 
+class GeneratedAsRowColumnConstraint(ColumnConstraintKind):
+    arg_types = {"start": False, "hidden": False}
```

Comment:
It seems to me that `start` is mandatory so this needs to become `True`

```suggestion
    arg_types = {"start": True, "hidden": False}
```